src/request.ts

```diff
@@ -17,6 +17,32 @@ import type {
   Response,
 } from "../src/types.ts";
 
+/**
+ * requestProxy converter.
+ * 
+ * @public
+ */
+export const requestProxy: Function = (req: Request) => {
+  return new Proxy(req, {
+    get: (target: any, name: string | number | symbol) => {
+      if (name === "body") return target.parsedBody ?? target.body;
+      return target[name];
+    },
+    set: (target: any, name: string | number | symbol, value: any): boolean => {
+      try {
+        if (name === "body") {
+          target.parsedBody = value;
+        } else {
+          target[name] = value;
+        }
+        return true;
+      } catch (_) {
+        return false;
+      }
+    },
+  });
+};
+
```

Comment:
In addition, it would be great to keep access to the original `body` property post parsing via a `raw` property.